plugins/lsp/default.nix

```diff
@@ -113,8 +113,7 @@ lib.nixvim.plugins.mkNeovimPlugin {
       type = types.bool;
       default = false;
       description = ''
-        Whether to enable LSP inlay-hints.
-        Only affects language servers with inlay-hints support.
+        Whether to enable LSP inlay-hints globally.
```

Comment:
This may be a good place to document how to do this per-lsp and/or per-buffer:

`````md
	To use inlay hints only on specific servers, you should call `vim.lsp.inlay_hint.enable` in your `on_attach` function:

	```lua
	on_attach = function(client, bufnr)
	  vim.lsp.inlay_hint.enable(true, { bufnr = bufnr })
	end
```
`````